src/vm/crst.h

```diff
@@ -296,7 +296,11 @@ friend class Crst;
 #endif
 
     union {
+#ifdef CROSSGEN_COMPILE
```

Comment:
This may rather be `CROSS_COMPILE`  so that it kicks for any kind of cross-compilation (debugger DAC), not just for crossgen.